.github/workflows/test.yml

```diff
@@ -213,10 +213,72 @@ jobs:
     - name: unit test
       run: sudo -E PATH="$PATH" -- make GOARCH=386 localunittest
 
+  fedora:
+    timeout-minutes: 30
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: lima-vm/lima-actions/setup@v1
+      id: lima-actions-setup
+
+    - uses: actions/cache@v4
+      with:
+        path: ~/.cache/lima
+        key: lima-${{ steps.lima-actions-setup.outputs.version }}
+
+    - name: "Start VM"
+      # --plain is set to disable file sharing, port forwarding, built-in containerd, etc. for faster start up
+      #
+      # CPUs: min(4, host CPU cores)
+      # RAM:  min(4 GiB, half of host memory)
+      # Disk: 100 GiB
+      run: limactl start --plain --name=default template://fedora
```

Comment:
lima autodetects mem and CPU, then?

Reply:
Yes
https://github.com/lima-vm/lima/blob/v1.0.6/pkg/limayaml/defaults.go#L144-L158